crates/game/pong/src/game.rs

```diff
@@ -19,25 +33,30 @@ pub struct PongGameState {
     pub ball_dir: Vec2,
 }
 
+pub fn random_ball_direction() -> f32 {
+    random_range(-FRAC_PI_4..FRAC_PI_4)
+}
+
 impl PongGame {
     pub fn new(player_one: PongPlayer, player_two: PongPlayer) -> Self {
-        let start_direction = random_range(0.0..(PI * 2.));
+        let mut ball_dir = Vec2::from_angle(random_ball_direction());
+        ball_dir.x *= -1.0;
 
         PongGame {
             player: (player_one, player_two),
             state: PongGameState {
                 score: (0, 0),
                 ball_pos: vec2(0.5, 0.5),
-                ball_dir: Vec2::from_angle(start_direction),
+                ball_dir,
             },
         }
     }
 
-    fn reset_ball(&mut self) {
-        let start_direction = random_range(0.0..(PI * 2.));
-
+    fn reset_ball(&mut self, direction: Direction) {
         self.state.ball_pos = vec2(0.5, 0.5);
-        self.state.ball_dir = Vec2::from_angle(start_direction);
+
+        self.state.ball_dir = Vec2::from_angle(random_ball_direction());
+        direction.orient_vec2(&mut self.state.ball_dir);
```

Comment:
I meant, that you would do something like that:
```suggestion
        self.state.ball_dir = Vec2::from_angle(random_ball_direction(&self.state.ball_dir));
```

So for right return between pi/4 and -pi/4 and for left between pi/4\*3 and pi/4\*5